templates/deno/.vscode/resolve_npm_imports.json

```diff
@@ -5,11 +5,11 @@
   "// Deno-only dependencies may be imported via URL imports (without using import maps).": "",
 
   "imports": {
-    "react": "https://esm.sh/react@18.0.0",
-    "react-dom": "https://esm.sh/react-dom@18.0.0",
-    "react-dom/server": "https://esm.sh/react-dom@18.0.0/server",
-    "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev@1.4.3/server-build",
+    "react": "https://esm.sh/react@17.0.2",
+    "react-dom": "https://esm.sh/react-dom@17.0.2",
+    "react-dom/server": "https://esm.sh/react-dom@17.0.2/server",
     "@remix-run/deno": "https://esm.sh/@remix-run/deno@1.4.3",
+    "@remix-run/dev/server-build": "https://esm.sh/@remix-run/dev@1.4.3/server-build",
     "@remix-run/react": "https://esm.sh/@remix-run/react@1.4.3"
```

Comment:
We should update the `@remix-run/*` versions to `1.5.0` so that it references a version where `@remix-run/deno` will be available

Reply:
Probably a good idea to update this each time we release

Reply:
This is implemented in #3225